src/Connections/FeedConnection.ts

```diff
@@ -95,10 +112,16 @@ export class FeedConnection extends BaseConnection implements IConnection {
                 url: this.feedUrl,
                 label: this.state.label,
             },
+            secrets: {
+                lastResults: [...this.lastResults.entriesDescending()].map(e => e[1]),
```

Comment:
I'm not sure about this use of QuickLRU as a circular buffer :P I guess it does the job, but I think it brings a bit more mental overhead than an array with an index tracker (or just array we push to, and shorten in at the front) would.

Reply:
(Or just a two-element array, if we only look back once).

Reply:
Yeah, fair point :)

Reply:
Done in [5fe3d4e](https://github.com/matrix-org/matrix-hookshot/pull/459/commits/5fe3d4e3d69d9ee361d4c7dec8e2f00a195e16c7)